server/openapi.yml

```diff
@@ -24,9 +24,48 @@ paths:
                 type: array
                 items:
                   $ref: "#/components/schemas/User"
+    post:
+      tags:
+        - user
+      summary: ユーザ作成API
+      description: ユーザを作成する
+      requestBody:
+        $ref: "#/components/requestBodies/UserBody"
+      responses:
+        201:
+          description: 追加されたユーザ情報
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/User"
+        400:
+          $ref: "#/components/responses/BadRequest"
 components:
+  responses:
+    BadRequest:
+      description: リクエストパラメーターに不備がある時のレスポンス
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/Error"
   schemas:
+    Error:
+      description: エラー情報
+      type: object
+      required:
+        - code
+        - message
+      properties:
+        code:
+          type: string
+          example: XXX
+          description: HTTPステータスコード
+        message:
+          type: string
+          example: エラーが発生しました
+          description: エラーメッセージ
```

Comment:
> エラーごとのスキーマは作らず，ここにexample書いた方がいいとおもう。なぜなら，`400`のエラーレスポンスを書く度にexample書くのは冗長になるから。OpenAPIの仕様書書く時には，冗長な箇所をできるだけ無くすことを意識した方がいいと思う。

exampleを抽象的に書いて共通化してみた

Reply:
やるならこんな感じ
```
    BadRequest:
      description: リクエストパラメーターに不備がある時のレスポンス
      content:
        application/json:
          schema:
            $ref: "#/components/schemas/Error"
          example:
            code: 400
            message: リクエストエラーが発生しました
```

`$ref`も上書きできるんやな

> Any sibling elements of a $ref are ignored. This is because $ref works by replacing itself and everything on its level with the definition it is pointing at. Consider this example:

https://swagger.io/docs/specification/using-ref/

Reply:
`BadRequest`の方に書く感じか